bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Image.java

```diff
@@ -1186,16 +1196,31 @@ void destroy () {
 
 private void destroyHandle () {
 	for (Long handle : zoomLevelToHandle.values()) {
-		if (type == SWT.ICON) {
-			OS.DestroyIcon (handle);
-		} else {
-			OS.DeleteObject (handle);
-		}
+		destroyHandle(handle);
 	}
 	zoomLevelToHandle.clear();
 	handle = 0;
 }
 
+@Override
+void destroyHandlesExcept(Set<Integer> zoomLevels) {
+	zoomLevelToHandle.entrySet().removeIf(entry -> {
+		if (!zoomLevels.contains(entry.getKey()) && entry.getKey() != DPIUtil.getZoomForAutoscaleProperty(initialNativeZoom)) {
+			destroyHandle(entry.getValue());
+            return true;
+        }
+        return false;
+	});
```

Comment:
Please properly format code (here and at other similar places).

Reply:
Done

Reply:
Fotmatting is still inconsistent.

Reply:
@amartya4256 I noticed that some of the new code uses spaces instead of tabs to indent. Maybe that's what Heiko means by "still inconsistent"?

Reply:
> I noticed that some of the new code uses spaces instead of tabs to indent. Maybe that's what Heiko means by "still inconsistent"?

Yes, seems to be a mixture of spaces and tabs. E.g., this:
![{DF3ACF41-FFEE-4B87-B72C-7560E8E0F39D}](https://github.com/user-attachments/assets/aa86acdc-e833-475e-973a-8b80919b6737)

@amartya4256 can you please adapt the inconsistent formatting at all remaining places?

Reply:
They seem to be done.

I'll mark this as resolved. If there are any open issues, they can be addressed in a follow-up PR.